packages/medusa/src/services/__tests__/claim.js

```diff
@@ -367,6 +367,16 @@ describe("ClaimService", () => {
       ).rejects.toThrow(`Cannot fulfill a claim without a shipping method.`)
     })
 
+    it("fails if claim is canceled", async () => {
```

Comment:
An example of these assertions that functionality rejects objects which are canceled